build.sbt

```diff
@@ -30,15 +31,16 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-sql" % sparkVersion.value % "provided" withSources(),
   "org.apache.spark" %% "spark-core" % sparkVersion.value % "provided" withSources(),
   "org.apache.spark" %% "spark-catalyst" % sparkVersion.value % "provided" withSources(),
-  "io.delta" %% "delta-core" % "0.6.1" % "provided" withSources(),
+  "io.delta" %% "delta-core" % deltaVersion % "provided" withSources(),
 
   // Test dependencies
   "org.scalatest" %% "scalatest" % "3.0.5" % "test",
   "org.mockito" %% "mockito-scala" % "0.4.0" % "test",
   "org.apache.spark" %% "spark-catalyst" % sparkVersion.value % "test" classifier "tests",
   "org.apache.spark" %% "spark-core" % sparkVersion.value % "test" classifier "tests",
   "org.apache.spark" %% "spark-sql" % sparkVersion.value % "test" classifier "tests",
-  "org.apache.spark" %% "spark-hive" % sparkVersion.value % "test" classifier "tests"
+  "org.apache.spark" %% "spark-hive" % sparkVersion.value % "test" classifier "tests",
+  "io.delta" %% "delta-core" % deltaVersion % "test"
```

Comment:
@sezruby @apoorvedave1 @pirz Can you compile/run tests inside InteillJ without this change (clear io.delta cache for ivy first)?

Reply:
@imback82 successfully compiled/ran tests without this change.

Reply:
even after clearing the ivy cache?

Reply:
![image](https://user-images.githubusercontent.com/12103644/103605890-ed350500-4ec9-11eb-9082-b1eb2aa05aa8.png)

Reply:
Yes I removed ~/.ivy2/cache and redownloaded all. If this change fixes the issue, then let's just merge it. It looks fine to me.